### PR TITLE
Metadata field security no projects alert

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes

Added an `Alert` that tells the user if there are no projects available to share with.  This could be do to the fact that they only have one project or that they are not a manager on any other projects.

## Related issue

N/A

## Checklist

Things for the developer to confirm they've done before the PR should be accepted:

*   [ ] ~CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change~. Will be added to final PR
*   [ ] ~Tests added (or description of how to test) for any new features~ Will be added to the final PR
*   [ ] ~User documentation updated for UI or technical changes~.Will be added to final PR

### CODE REVIEW

**Review by deepsidhu85 — APPROVED** [review]
This works great and looks good. Will merge once we get dev merged up into our metadata field security stuff

File: src/main/webapp/resources/js/pages/projects/share/ShareProject.jsx
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/projects/share/ShareProject.jsx:4]: Don't think this is being used and can be removed?

File: src/main/webapp/resources/js/pages/projects/share/ShareProject.jsx
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/projects/share/ShareProject.jsx:13]: Can we get the projects from the state here instead of passing it in?

File: src/main/webapp/resources/js/pages/projects/share/index.js
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/projects/share/index.js:158]: I think we can change this to `projects?.length === 0`

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
Looks great and working as expected. Just a few small comments below otherwise good to go

File: src/main/webapp/resources/js/pages/projects/share/ShareProject.jsx
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/share/ShareProject.jsx:4]: Removed in 6d21c776a1f9698293d9d200534ff09576a74775

File: src/main/webapp/resources/js/pages/projects/share/ShareProject.jsx
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/share/ShareProject.jsx:13]: Ignoring :)

File: src/main/webapp/resources/js/pages/projects/share/index.js
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/share/index.js]: I like this, nice find :) updated in 483fa813c506a861710628ef8da40288bc501821

**Review by deepsidhu85 — APPROVED** [review]

**Pull request merged**
